docs/querying/querying.md

```diff
@@ -117,9 +117,12 @@ If a query fails, you will get an HTTP 500 response containing a JSON object wit
   "host" : "druid1.example.com:8083"
 }
 ```
+The HTTP response code returned depends on the type of query failure. For timed out queries, an HTTP 504 response code is returned.
 
 If a query request fails due to being limited by the [query scheduler laning configuration](../configuration/index.md#broker), an HTTP 429 response with the same JSON object schema as an error response, but with `errorMessage` of the form: "Total query capacity exceeded" or "Query capacity exceeded for lane 'low'".
 
+For every other type of query failures, an HTTP 500 response code is returned.
```

Comment:
The other missing response codes here are 401 and 403 which can be returned for auth failures. Would you please add them as well?

Reply:
Good catch. Added